src/test/java/socialite/model/person/RemarkTest.java

```diff
@@ -0,0 +1,31 @@
+package socialite.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+public class RemarkTest {
+
```

Comment:
Actually in hindsight I realised that maybe we should include a test to make sure remark is not an empty string containing only whitespaces i.e. `"   "`. We can include this in future commits.